<UI version="4.0" >
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass" >
	 <property name="geometry">
		 <rect>
			 <x>0</x>
			 <y>0</y>
			 <width>800</width>
			 <height>636</height>
		 </rect>
	 </property>
	 <property name="windowTitle">
		 <string>MainWindow</string>
	 </property>
	 <widget class="QWidget" name="centralwidget">
		 <widget class="QCustomPlot" name="plotWidget" native="true">
			 <property name="geometry">
				 <rect>
					 <x>10</x>
					 <y>10</y>
					 <width>771</width>
					 <height>511</height>
				 </rect>
			 </property>
			 <property name="styleSheet">
				 <string notr="true">background: #fff;</string>
			 </property>
		 </widget>
		 <widget class="QPushButton" name="drawButton">
			 <property name="geometry">
				 <rect>
					 <x>10</x>
					 <y>540</y>
					 <width>151</width>
					 <height>41</height>
				 </rect>
			 </property>
			 <property name="font">
				 <font>
					 <pointsize>16</pointsize>
				 </font>
			 </property>
			 <property name="text">
				 <string>DRAW</string>
			 </property>
		 </widget>
		 <widget class="QLineEdit" name="firstCoefEdit">
			 <property name="geometry">
				 <rect>
					 <x>280</x>
					 <y>540</y>
					 <width>111</width>
					 <height>41</height>
				 </rect>
			 </property>
			 <property name="font">
				 <font>
					 <pointsize>20</pointsize>
				 </font>
			 </property>
		 </widget>
		 <widget class="QLineEdit" name="secondCoefEdit">
			 <property name="geometry">
				 <rect>
					 <x>480</x>
					 <y>540</y>
					 <width>111</width>
					 <height>41</height>
				 </rect>
			 </property>
			 <property name="font">
				 <font>
					 <pointsize>20</pointsize>
				 </font>
			 </property>
		 </widget>
		 <widget class="QLabel" name="label">
			 <property name="geometry">
				 <rect>
					 <x>250</x>
					 <y>540</y>
					 <width>31</width>
					 <height>41</height>
				 </rect>
			 </property>
			 <property name="font">
				 <font>
					 <pointsize>20</pointsize>
				 </font>
			 </property>
			 <property name="text">
				 <string>k :</string>
			 </property>
		 </widget>
		 <widget class="QLabel" name="label_2">
			 <property name="geometry">
				 <rect>
					 <x>450</x>
					 <y>540</y>
					 <width>31</width>
					 <height>41</height>
				 </rect>
			 </property>
			 <property name="font">
				 <font>
					 <pointsize>20</pointsize>
				 </font>
			 </property>
			 <property name="text">
				 <string>b :</string>
			 </property>
		 </widget>
		 <widget class="QLabel" name="label_3">
			 <property name="geometry">
				 <rect>
					 <x>610</x>
					 <y>540</y>
					 <width>191</width>
					 <height>41</height>
				 </rect>
			 </property>
			 <property name="font">
				 <font>
					 <pointsize>15</pointsize>
				 </font>
			 </property>
			 <property name="styleSheet">
				 <string notr="true">color: blue;</string>
			 </property>
			 <property name="text">
				 <string>y = k * sin(x + b)</string>
			 </property>
		 </widget>
	 </widget>
	 <widget class="QMenuBar" name="menubar">
		 <property name="geometry">
			 <rect>
				 <x>0</x>
				 <y>0</y>
				 <width>800</width>
				 <height>21</height>
			 </rect>
		 </property>
	 </widget>
	 <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
	 <customwidget>
		 <class>QCustomPlot</class>
		 <extends>QWidget</extends>
		 <header location="global">qcustomplot.h</header>
		 <container>1</container>
	 </customwidget>
 </customwidgets>
 <layoutDefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources>
   <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</UI>
